### DISCUSSION TITLE
ToonSquid Monthly Jams/Challenges Starting on November.

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#672] [General]
I have decided to start monthly challenge for ToonSquid just like Callipeg. If you are interested than get ready.
![Screenshot 2024-10-16 081808](https://github.com/user-attachments/assets/9da9cdec-6bcd-48a5-be24-a60e300867b0)

**Comment by Lioncub901** [comment]
Link: https://x.com/ToonsquidCommu/status/1846300246697009528

**Reply by TempleLionStudios** [reply]
Id love to join. Im in many of callipegs challenge videos. 

Maybe we should sync up, since we do monthly challenges on the "ipad animators"-discord server, which focuses on ToonSquid 😊

Can you dm me on discord? 🏆

https://discord.gg/qeF7QNAd2H

We could release videoes with peoples animations!

**Reply by Lioncub901** [reply]
I contacted the main mod on the discord and the mod said I should make a separate contest from their one.

**Reply by TempleLionStudios** [reply]
Maybe you are confusing several people. Since I run that server I can see we haven't had that conversation 😊
